hasjob/views/index.py

```diff
@@ -776,3 +776,18 @@ def logoimage(domain, hashid):
 @app.route('/search')
 def search():
     return redirect(url_for('index', **request.args))
+
+
+@app.route('/service-worker.js', methods=['GET'])
+def sw():
+    return app.send_static_file('service-worker.js')
+
+
+@app.route('/workbox-sw.prod.v2.1.2.js', methods=['GET'])
+def workbox():
+    return app.send_static_file('workbox-sw.prod.v2.1.2.js')
```

Comment:
What is `v2.1.2`?

Reply:
`workbox-webpack-plugin` generates the service worker that import's Workbox's service worker library
https://developers.google.com/web/tools/workbox/get-started/webpack#register

Reply:
So v2.1.2 is a static path that won't change? Or it's linked to the webpack version? If latter, we should avoid a dependency on having to change this line every time webpack changes.